src/AppShed/Extensions/StorageBundle/Controller/WriteController.php

```diff
@@ -52,6 +52,17 @@ public function appshedAction(Request $request)
         $store = $view->getStore();
 
         $data = Remote::getRequestVariables();
+
+        $systemFields = $this->container->getParameter('data_system_fields');
+        foreach ($data as $key => $record) {
+            if (in_array($key, $systemFields)) {
+                unset($data[$key]);
```

Comment:
replace it to   continue;  
http://php.net/manual/en/control-structures.foreach.php
As foreach relies on the internal array pointer, changing it within the loop may lead to unexpected behavior.